pages/parent-enrollment.tsx

```diff
@@ -0,0 +1,76 @@
+import Wrapper from "@components/SDCWrapper";
+import { Flex, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseButton } from "@components/CloseButton";
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+    const [selectedChild, setSelectedChild] = useState<string>("");
+
+    return (
+        <Wrapper>
+            <Flex justifyContent="flex-end">
+                {/* navigate to browse programs page instead of going back */}
+                <CloseButton href="/"></CloseButton>
+            </Flex>
+            <Center>
+                <Text align="center" mt="15px" fontWeight="700" fontSize="36px">
+                    Program Registration
+                </Text>
+            </Center>
+            <Center>
+                <Text pb="55px" align="center" mt="30px">
+                    Who would you like to register for{" "}
```

Comment:
I think the `{" "}` might be there by mistake?

Reply:
I think this makes sure there is a space between the for and Building Bridges because of the bold tag